src/cloud/ops.rs

```diff
@@ -16,8 +16,8 @@ use crate::print::{self, echo, err_marker, Highlight};
 use crate::question;
 use crate::table::{self, Cell, Row, Table};
 
-const INSTANCE_CREATION_WAIT_TIME: Duration = Duration::from_secs(5 * 60);
-const INSTANCE_CREATION_POLLING_INTERVAL : Duration = Duration::from_secs(1);
+const INSTANCE_OPERATION_WAIT_TIME: Duration = Duration::from_secs(5 * 60);
```

Comment:
```suggestion
const OPERATION_WAIT_TIME: Duration = Duration::from_secs(5 * 60);
```
When you'll have to split operations into instance and non-instance, you can rename constant.